test/ping/ping.test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, test } from "mocha";
+import { server } from "../../src/app";
```

Comment:
Das Problem liegt darin wie du testest hier importiertst du deine app und die ist ja dependent auf den environment varia blen, da du ja aber hier keine env variablen gesetzt (und die nicht gesetzt werden in der Pipeline weils da keine .env Datei gibt) hast ist process.env.PORT undefined deshalb failed der test

Reply:
Es ist wahrscheinlich auch Auslegungssache, ob das ein Unit-Test oder ein Integration-Test ist weil du dir ja den ganzen server aufziehst und requests testest, statt eine kleine Unit, die Unterhaltung hatten wir ja auch schon Mal mit Tielsch weil Fynn und ich das im Kino genauso gemacht hatten, kann man sich aber wahrscheinlich drüber streiten. Die ping route ist ja jetzt noch nicht so ein großer Stück Code, spannend wird es erst wenn es an die komplexeren Routes geht @HaseFlrn